demo/k8s-validating-admission-policy/README.md

```diff
@@ -1,10 +1,12 @@
-This is a demo of a prototype-stage feature and is subject to change.
+> [!WARNING]
+> This is a demo of a prototype-stage feature and is subject to change.
 
-The demo will not work unless the `--experimental-enable-k8s-native-validation`` is
-set. Please set `--validate-template-rego` to `false` if using Gatekeeper version 3.13.1+ but before 3.16.0.
+## Pre-requisites
 
-Note that the contents of the constraint template have changed since cutting
-Gatekeeper's v3.13.0 release. To try this with the development build of
```

Comment:
there is a bit nuance here that's hard to explain since `variables` were not supported in 3.13. just to simplify, i changed it to min v3.14.